lib/src/ConfigLoader.cc

```diff
@@ -128,6 +129,46 @@ ConfigLoader::ConfigLoader(const std::string &configFile)
         }
     }
 }
+ConfigLoader::ConfigLoader(const std::string &configFile, ConfigSyntax syntax)
+{
+    if (os_access(drogon::utils::toNativePath(configFile).c_str(), 0) != 0)
+    {
+        std::cerr << "Config file " << configFile << " not found!" << std::endl;
```

Comment:
@an-tao I think this should be LOG_FATAL?

Reply:
done